paymentsheet/src/test/java/com/stripe/android/common/analytics/experiment/LogLinkGlobalHoldbackExposureTest.kt

```diff
@@ -174,9 +178,11 @@ class LogLinkGlobalHoldbackExposureTest {
         assertEquals(lookupCall.email, TestFactory.LINK_CONFIGURATION.customerInfo.email!!)
 
         val exposureCall = eventReporter.experimentExposureCalls.awaitItem()
-        assertTrue(exposureCall.experiment is LoggableExperiment.LinkGlobalHoldback)
-        assertEquals(exposureCall.experiment.group, "holdback")
-        assertTrue((exposureCall.experiment.isReturningLinkConsumer))
+        val experiment = exposureCall.experiment
+        assertTrue(experiment is LoggableExperiment.LinkGlobalHoldback)
+        assertThat(experiment).isInstanceOf(LoggableExperiment.LinkGlobalHoldback::class.java)
```

Comment:
Nit: These are asserting the same thing.

Reply:
https://github.com/stripe/stripe-android/pull/10565/commits/0878827c38487fd2e3b95ad8cb6caa3a951dd648